CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* BraintreeDataCollector
```

Comment:
Do we know what kind of warning a merchant would get from the Goog? I wonder if it's worth calling it out specifically for future us if merchants come in asking about it or if we will just remember it's a Kount thing. 🤷

Reply:
I'm not sure there's supposed to be a way to check by uploading the library in the Play console but I don't think it exists yet. That might be a good follow up question for them.